sdk/storage/azure-storage-queue/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Release History
 
-## 12.1.6 (Unreleased)
+## 12.1.6 (2021-04-20)
+**Fixes**
+- Make `AccountName`, `AccountKey` etc. in conn_str case insensitive
+- Fixed unclosed ThreadPoolExecutor (#8955)
+- Optimized memory usage for datalake file uploads large chunks (#16890)
```

Comment:
this is datalake related?

Reply:
hahaha read the description again, it's copied from your pr

Reply:
No I mean this is a datalake change but right now its under queues changelog

Reply:
hahahaha you are so right! I don't know why I paste it here